test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/IntegrationTest/otel-collector-config.yaml

```diff
@@ -16,13 +16,39 @@ receivers:
       grpc:
         endpoint: 0.0.0.0:5317
         tls:
-          cert_file: /cfg/otel-collector.crt
-          key_file: /cfg/otel-collector.key
+          cert_file: /cfg/certs/otel-test-server-cert.pem
+          key_file: /cfg/certs/otel-test-server-key.pem
       http:
         endpoint: 0.0.0.0:5318
         tls:
-          cert_file: /cfg/otel-collector.crt
-          key_file: /cfg/otel-collector.key
+          cert_file: /cfg/certs/otel-test-server-cert.pem
+          key_file: /cfg/certs/otel-test-server-key.pem
+  otlp/untrustedtls:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:6317
+        tls:
+          cert_file: /cfg/certs/otel-untrusted-collector-cert.pem
+          key_file: /cfg/certs/otel-untrusted-collector-key.pem
+      http:
+        endpoint: 0.0.0.0:6318
+        tls:
+          cert_file: /cfg/certs/otel-untrusted-collector-cert.pem
+          key_file: /cfg/certs/otel-untrusted-collector-key.pem
+  otlp/mtls:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:7317
+        tls:
+          cert_file: /cfg/certs/otel-test-server-cert.pem
+          key_file: /cfg/certs/otel-test-server-key.pem
+          client_ca_file: /cfg/certs/otel-test-ca-cert.pem
+      http:
+        endpoint: 0.0.0.0:7318
+        tls:
+          cert_file: /cfg/certs/otel-test-server-cert.pem
+          key_file: /cfg/certs/otel-test-server-key.pem
+          client_ca_file: /cfg/certs/otel-test-ca-cert.pem
```

Comment:
These are not yet exercised by the integration tests. @sandy2008 is it your plan to follow up with additional PRs introducing the new functionality? If so, I'm fine leaving these here.